packages/functional-tests/lib/targets/remote.ts

```diff
@@ -7,12 +7,11 @@ export abstract class RemoteTarget extends BaseTarget {
     password: string,
     options?: any
   ): Promise<Credentials> {
-    const creds = await this.auth.signUp(email, password);
+    const creds = await this.auth.signUp(email, password, options || {});
```

Comment:
The options are optional on this.auth.signUp as well, so I don't think we need the `|| {}`. I can definitely see how not passing options in some cases causes issues, so I think that passing options in makes sense.